CMakeLists.txt

```diff
@@ -203,25 +213,45 @@ if("${DDC_BUILD_KERNELS_SPLINES}")
         find_package(KokkosKernels REQUIRED)
     endif()
 
-    target_link_libraries(DDC INTERFACE Kokkos::kokkoskernels)
-    target_compile_definitions(DDC INTERFACE "DDC_BUILD_KERNELS_SPLINES")
+    add_library(ddc_splines INTERFACE)
+    add_library(DDC::splines ALIAS ddc_splines)
+    install(FILES cmake/FindLAPACKE.cmake DESTINATION lib/cmake/DDC)
+    install(TARGETS ddc_splines EXPORT DDCTargets)
+    target_compile_definitions(ddc_splines INTERFACE "DDC_BUILD_KERNELS_SPLINES")
+    target_include_directories(ddc_splines SYSTEM INTERFACE ${LAPACKE_INCLUDE_DIRS})
+    target_link_libraries(
+        ddc_splines
+        INTERFACE ddc Ginkgo::ginkgo Kokkos::kokkos Kokkos::kokkoskernels ${LAPACKE_LIBRARIES}
+    )
+
+    ### DDC legacy target
+    target_link_libraries(DDC INTERFACE ddc_splines)
 endif()
 
 ## The PDI wrapper
 
 if("${DDC_BUILD_PDI_WRAPPER}")
-    add_library(PDI_Wrapper INTERFACE)
-    target_compile_features(PDI_Wrapper INTERFACE cxx_std_17)
+    add_library(ddc_pdi INTERFACE)
+    add_library(DDC::pdi ALIAS ddc_pdi)
+    install(TARGETS ddc_pdi EXPORT DDCTargets)
+    target_compile_definitions(ddc_pdi INTERFACE "DDC_BUILD_PDI_WRAPPER")
+    target_compile_features(ddc_pdi INTERFACE cxx_std_17)
     target_include_directories(
-        PDI_Wrapper
+        ddc_pdi
         INTERFACE
             "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>"
             "$<INSTALL_INTERFACE:include>"
     )
-    target_link_libraries(PDI_Wrapper INTERFACE DDC::DDC PDI::PDI_C)
-    target_compile_definitions(PDI_Wrapper INTERFACE "DDC_BUILD_PDI_WRAPPER")
+    target_link_libraries(ddc_pdi INTERFACE ddc PDI::PDI_C)
+
+    ### PDI_Wrapper legacy target
+    add_library(PDI_Wrapper INTERFACE)
```

Comment:
Why not remove `PDI_Wrapper` and directly alias `DDC::PDI_Wrapper` to `ddc_pdi` in `DDCConfig.cmake`?

Reply:
I would prefer to deprecate it first for a smooth transition.

Reply:
Actually cmake also has a tool for that, https://cmake.org/cmake/help/latest/prop_tgt/DEPRECATION.html, I would like to give a try

Reply:
done in 3bda09adbfa6a1c4decec528e060119fbf9aa21b

Reply:
does that mean that you have users downstream that depend on a non-namespaced `PDI_Wrapper`?

Reply:
It should not be possible because of the namespace. I think the question is whether we want to break their code with this PR or warn them to use the new DDC::pdi target ?

Reply:
I now understand, you need an `INTERFACE` target to add it the `DEPRECATION` property 👍